includes/definitions.inc.php

```diff
@@ -888,8 +888,8 @@
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
 $os = 'fabos';
-$config['os'][$os]['text']             = 'Brocade FabricOS';
-$config['os'][$os]['type']             = 'network';
+$config['os'][$os]['text']             = 'FabricOS';
+$config['os'][$os]['type']             = 'storage';
```

Comment:
I'm not overly fussed by this change but if it was up to me then I wouldn't change it to storage as the device provides no storage facilities.